### PR TITLE
add new client troubleshooting section on poor performance in U-boot

### PR DESCRIPTION
**Pull request by mirzak**
Changelog: None

Signed-off-by: Mirza Krak <mirza.krak@northern.tech>

### CODE REVIEW

**Comment by mirzak** [comment]
Also my editor seems to have cleared some trailing white spaces :)

**Comment by estenberg** [comment]
Very clear!

Regarding the location, my intent at least was that "Mender client" troubleshooting would be board-agnostic. It's assuming the board integration is working.

I think this would fit better under https://docs.mender.io/2.0/troubleshooting/running-yocto-project-image, there are other sections about GRUB and U-Boot there. We might also want to rename that doc page to "Boot" or something similar, now that we have more than just Yocto.

File: 201.Troubleshooting/03.Mender-Client/docs.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ Each TLS certificate has a validity period, *Not Before* and *Not After*, and th
 the current time is outside this range.
 
 Most commonly this is caused by **incorrect time setting at the device** which runs the Mender client. Check this by
-running `date` at the device, and make sure it is correct. Consult the section on [Correct clock](../../devices/general-system-requirements#correct-clock) 
+running `date` at the device, and make sure it is correct. Consult the section on [Correct clock](../../devices/general-system-requirements#correct-clock)
```
Comment (drewmoseley) [inline 201.Troubleshooting/03.Mender-Client/docs.md]: running `date` on the device

**Comment by mirzak** [comment]
Changed section for troubleshooting and updated wording based on your comment @drewmoseley

**Pull request merged**